dotnet/samples/04.ai.d.chainedActions.listBot/README.md

```diff
@@ -0,0 +1,159 @@
+# Microsoft Teams Conversational Bot with AI: Teams Chef
+
+## Summary
+
+ListBot: Your Ultimate List Management Companion. Powered by advanced AI capabilities, this innovative bot is designed to streamline task list. With the ability to create, update, and search lists and tasks, the ListBot offers a seamless and efficient solution to help you stay on top of your to-do's and maximize productivity. Experience the ease of list management like never before, as the ListBot harnesses the power of AI to simplify your workflow and bring order to your daily tasks and showcases the action chaining capabilities.
+
+It shows SDK capabilities like:
+
+<details open>
+    <summary><h3>Bot scaffolding</h3></summary>
+    Throughout the 'Program.cs' file you'll see the scaffolding created to run a bot, e.g. storage, authentication, and conversation state.
+</details>
+
+<details open>
+    <summary><h3>Prompt engineering</h3></summary>
+The `Prompts/Chat/`, `Prompts/ChatGPT/` and `Prompts/Summarize` directories have descriptive prompt engineering that, in plain language, instructs GPT how the bot should conduct itself at submit time. For example, in `Prompts/ChatGPT/skprompt.txt`:
+
+#### skprompt.txt
```

Comment:
This is the `Chat` prompt? Above line says ChatGPT.

Reply:
Oh yes, updated.